util/intstr/parametrizable.go

```diff
@@ -0,0 +1,55 @@
+package intstr
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// These are utility functions when using IntOrString to hold either an Int or an Argo Variable
+func Int(is *intstr.IntOrString) (*int, error) {
+	if is == nil {
+		return nil, nil
+	}
+	if is.Type == intstr.String {
```

Comment:
just use `is.IntValue()`?

Reply:
my guts say we should just swallow errors in the string and return zero

Reply:
How do we tell apart an error from an explicit `0`?